Sources/Plasma/PubUtilLib/plNetGameLib/Private/plNglAuth.cpp

```diff
@@ -1249,7 +1248,7 @@ enum {
 
 static bool                         s_running;
 static std::recursive_mutex         s_critsect;
-static LISTDECL(CliAuConn, link)    s_conns;
+static CliAuConn* s_conn = nullptr;
 static CliAuConn *                  s_active;
```

Comment:
Are there ever any times that `s_conn` and `s_active` will not be the same? I'm wondering if we could just use `s_active` instead...

Reply:
That's what I wanted to do initially, but unfortunately the two aren't exactly the same. `s_conns`/`s_conn` is always set before `AsyncSocketConnect`, whereas `s_active` is only set later once the connection is fully established (e. g. for the auth server only after a `Auth2Cli_ClientRegisterReply` is received).

This logic can probably be simplified somehow so that it only uses one variable, but I didn't want to do anything too risky as part of the legacy data structure cleanup.

Reply:
That seems like a good approach. 👍